README.md

```diff
@@ -42,27 +42,29 @@ build succeeded.
 Build finished. The HTML pages are in build/html.
 ```
 
+# Tests
+Ideally run these within a virtual environment as to not globally install pytest.
 ## Test Setup
 
 Running the setup script will:
 
 1. Setup the test postgres tables
 
 ```
-❯ python psqlgraph/setup_psql_graph.py
+❯ python setup.py install
+❯ python test/setup_test_psqlgraph.py
```

Comment:
Are these instructions actually correct? I thought `setup.py install` built an egg and installed the egg in the system's site packages folder. But in the next steps, you're assuming a directory and directly executing the files. So what's the point of installing the egg in site packages?

Reply:
Because tests/setup_test_psqlgraph.py requires having sqlalchemly. I think the install/egg building might be superfluous and `python setup develop` might be better suited.  

Came in as a user ticket.